app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -2358,12 +2365,15 @@ public void onClick(
                     case -1:
                     case AppCompatDelegate.MODE_NIGHT_NO:
                         setNightMode(AppCompatDelegate.MODE_NIGHT_YES);
+                        app.makeNightThemeDefault(true);
                         break;
                     case AppCompatDelegate.MODE_NIGHT_YES:
```

Comment:
NIT: As the default and this is same, we can remove the block (Upto you how you want to do it) 
```
setNightMode(AppCompatDelegate.MODE_NIGHT_NO);
app.makeNightThemeDefault(false);
```